src/electron/flux/types/android-setup-state-machine-types.ts

```diff
@@ -11,14 +11,15 @@ export type AndroidSetupStateMachine = StateMachine<AndroidSetupStepId, AndroidS
 export type AndroidSetupStepTransitionCallback = (nextStep: AndroidSetupStepId) => void;
 
 export type AndroidSetupStoreCallbacks = {
-    stepTransition: AndroidSetupStepTransitionCallback;
     setSelectedDevice: (device: DeviceInfo) => void;
     setAvailableDevices: (devices: DeviceInfo[]) => void;
+
```

Comment:
Is this extra newline in the middle of the type intentional?

Reply:
The code was that way when I found it. But I have now removed it.